client/src/App.jsx

```diff
@@ -23,10 +24,10 @@ function App() {
           <Route path="/teacherreg" element={<TeacherRegistration />} />
           <Route path="/studentlogin" element={<StudentLogin />} />
           <Route path="/teacherlogin" element={<TeacherLogin />} />
+          <Route path="/coursepage" element={<CoursePage />} />
```

Comment:
might want to rename the path to `/student/courses` to make it more obvious of which course page is being accessed